### PR TITLE
LibWeb: Updates to increase test runnability of WPT

### PR DESCRIPTION
**Pull request by ADKaster**
- Stub out WebDriver deserialize as a proxy capability AO
- Unblock port 9000
- Implement document.elementsFromPoint (not to be confused with elementFromPoint)

Brings the WPT infrastructure/ directory to the following (with SSL cert checking disabled)

```
Ran 148 tests finished in 617.0 seconds.
  • 37 ran as expected. 0 tests skipped.
  • 39 tests had errors unexpectedly
  • 21 tests failed unexpectedly
  • 26 tests timed out unexpectedly
  • 32 tests had unexpected subtest results
```

On master it's not actually possible to run all these tests b/c of the proxy capability issue.

Before the elementsFromPoint commit, the results are like so:

```
Ran 148 tests finished in 732.1 seconds.
  • 36 ran as expected. 0 tests skipped.
  • 43 tests had errors unexpectedly
  • 21 tests failed unexpectedly
  • 29 tests timed out unexpectedly
  • 25 tests had unexpected subtest results
```

### CODE REVIEW

File: Userland/Libraries/LibWeb/DOM/Document.cpp [hunk 1]
```diff
@@ -3767,4 +3767,46 @@ Element const* Document::element_from_point(double x, double y)
...
+                if (auto* dom_node = result->dom_node(); dom_node && dom_node->is_element())
+                    sequence.append(*static_cast<Element*>(dom_node));
+            }
+            return Painting::TraversalDecision::Continue;
+        });
+    }
+
+    // 4. If the document has a root element, and the last item in sequence is not the root element,
+    //    append the root element to sequence.
+    if (auto* document_root_element = first_child_of_type<Element>(); document_root_element)
```
Comment (trflynn89) [inline Userland/Libraries/LibWeb/DOM/Document.cpp:3805]: :thinking: can just use `document_element()`, right?

File: Userland/Libraries/LibWeb/DOM/Document.cpp [hunk 2]
```diff
@@ -3767,4 +3767,46 @@ Element const* Document::element_from_point(double x, double y)
...
+        sequence.append(*document_root_element);
```
Comment (trflynn89) [inline Userland/Libraries/LibWeb/DOM/Document.cpp:3806]: Missing the check for "and the last item in sequence is not the root element"

**Review by trflynn89 — COMMENTED** [review]
Can we add in-tree tests for the new API?

File: Userland/Libraries/LibWeb/DOM/Document.cpp (hunk 1 above)
Comment (ADKaster) [reply Userland/Libraries/LibWeb/DOM/Document.cpp:3805]: it says in the CSS-DISPLAY-4 spec that the `root element` is

> The [element](https://drafts.csswg.org/css-display-4/#elements) at the root of the [document tree](https://dom.spec.whatwg.org/#concept-document-tree). In a document tree produced under the DOM, this is the [document element](https://dom.spec.whatwg.org/#document-element); in HTML it is the [html](https://html.spec.whatwg.org/multipage/semantics.html#the-html-element) element. [[DOM]](https://drafts.csswg.org/css-display-4/#biblio-dom) [[HTML]](https://drafts.csswg.org/css-display-4/#biblio-html)

 https://drafts.csswg.org/css-display-4/#root-element
 
 So it's... either the document element, or the html element?
 
 I honestly just copied this from the function above, so I'm not sure if it's correct or not either way...

Comment (ADKaster) [reply Userland/Libraries/LibWeb/DOM/Document.cpp:3805]: ,,, oh that's literally what `document_element()` does. it calls `first_child_of_type<Element>()` :sweat_smile:

**Pull request merged**